src/reducers/index.test.ts

```diff
@@ -19,6 +19,18 @@ const gameInfo: GameInfo = {
   screenShoot: image,
```

Comment:
Can you fix: ~screenShoot~ -> screenShot everywhere?

Reply:
oops :)
well, almost everywhere: you can't fix it in the db itself, sadly there it must remain screenShootImageId
:)